xml/deployment_configuration.xml

```diff
@@ -318,15 +320,109 @@ ssh_authorized_keys:
      will be prompted to change the default password at the first login.
     </para>
    </sect3>
-   <sect3 xml:id="user-data.config.adding.custom.repository">
+      <sect3 xml:id="user-data.config.adding.custom.repository">
     <title>Adding Custom Repository</title>
-    <para>
-     You can add a custom software repository to your system by using the
-     <literal>zypp_repos</literal> option:
-    </para>
-<screen>zypp_repos:
-  "Update Repo":
-      baseurl: http://my.example.com/CAASP/repo/SUSE-CAASP-1.0-CUSTOM/</screen>
+    <para> You can add a custom software repository to your system by using the
+      <literal>zypp_repos</literal> option: </para>
+    <screen>
+        zypper:
+          repos:
+            - id: opensuse-oss
+              name: os-oss
+              baseurl: http://my.example.com/repo/SUSE-CAASP-&productnumber;-CUSTOM/
+              enabled: 1
+              autorefresh: 1
+            - id: opensuse-oss-update
+              name: os-oss-up
+              baseurl: http://my.example.com/repo/SUSE-CAASP-&productnumber;-CUSTOM/update
+      </screen>
```

Comment:
The indention of the &lt;/screen&gt; tag is not correct, it should match the opening tag. Because of the 1:1 display, you might want to not indent these two tags *at all* (at least this is what daps-xmlformat does).